### DISCUSSION TITLE
Droplets Coalescence

### DISCUSSION THREAD

**Discussion by nguyen2504to** [Colvars/colvars#653] [Q&A] [answered]
Hello,
I want to create the colvars file and apply WHAM for the case of 2 Droplets Coalescence. I created 2 droplets and made the coalescence by decreasing the distance between two COMs of each particle (for example, from 30 to 0)

Question:

1/ Can I apply the Colvar package of Lammps to calculate the PMF? I tried to apply by making 1 droplet as Ref, another as Main and setting the initial center, the target center is 0. However, I can not give the fine overlap of the histogram. What do you think I should do?
2/ When I have the histogram results, I use the WHAM for calculating the energy, and I get the "-nan" result. Would you happen to know how to fix this error?

Thank you.

**Comment by jhenin** [comment, marked as answer]
Hello,

Yes, you can use Colvars to do this. However, this seems more of an issue with how to set up and optimize Umbrella Sampling simulations than something specific to Colvars. If there is no overlap, then that explains the NaN result in WHAM. You may need additional windows to fill the gap.

Jérôme

**Comment by nguyen2504to** [comment]
Hi Jérôme,

When two droplets start to coalesce, the combination is too fast. That is why I can not apply to the US to have the fine overlap. Please show me which command can keep the atom from moving at each window.

Thanks

**Comment by jhenin** [comment]
It sounds like you may be using the wrong collective variable for this process.

**Comment by nguyen2504to** [comment]
I use the **distanceZ** to control the coalescence by decreasing the distance between two COMs of each particle.  I tried to apply by making 1 droplet as Ref, another as Main, and setting the initial center, the target center as 0.

**Comment by giacomofiorin** [comment]
Hello Thao, if there is significant mixing between the two droplets it may be difficult to define a CV based on the centers of mass.

**Comment by giacomofiorin** [comment]
@nguyen2504to Hope that the following simulations were more fruitful. Closing this, feel free to reopen if there is a technical issue with the code.
